File: 231A - Team/Team.cs
using System;

class Team
{
    static void Main()
    {
        int max = int.Parse(Console.ReadLine());
        int canSolve = 0;

        for (int i = 0; i < max; i++)
        {
            int count = 0;
            String str = Console.ReadLine();

            foreach (char c in str)
                count += (c == '1') ? 1 : 0;

            canSolve += (count >= 2) ? 1 : 0;
        }

        Console.Write(canSolve);
    }
}

File: 231A - Team/Team.py
max = int(input())
canSolve = 0

for i in range(max):
    count = 0
    line = input()

    for c in line:
        if (c == '1'):
            count += 1
    if (count >= 2):
        canSolve += 1

print(canSolve)